core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestService.java

```diff
@@ -127,6 +127,7 @@ public VersionRestService() {
 				.setDescription(input.getDescription())
 				.setEffectiveTime(input.getEffectiveTime())
 				.setForce(input.isForce())
+				.setCommitComment(input.getCommitComment())
```

Comment:
Use the `com.b2international.snowowl.core.rest.domain.ResourceRequest<T>` wrapper instead of using your custom field.